src/nccl_ofi_rdma.cpp

```diff
@@ -2653,6 +2519,25 @@ static int test(nccl_net_ofi_req_t *base_req, int *done, int *size)
 		ret = ofi_process_cq(ep);
 		if (OFI_UNLIKELY(ret != 0))
 			goto exit;
+
+		rdma_req_send_data_t *send_data = get_send_data(req);
```

Comment:
this needs a comment.  And I'd really like to figure out how not to have all this code in the test() critical path of a busy poll.